README.md

```diff
@@ -34,10 +34,13 @@ Install `fixed-data-table-2` using npm.
 ```shell
 npm install fixed-data-table-2
 ```
-Add the default stylesheet `dist/fixed-data-table.css`, then import it into any module.
+Add the default stylesheet `dist/fixed-data-table.css` using a link tag or import it with a CSS module.
 
 ### Basic Example
-
+Note this example demonstrates typical FDT usage, but won't work out of the box.  
```

Comment:
This seems excessive. Our library should be targeted towards normal react devs. If someone doesn't know how to compile JSX and how to set up a react app probably isn't ready to use this.